src/kartograf/filters/ring_changes.py

```diff
@@ -162,3 +163,55 @@ def get_atom_ring_hybridization_map(rdmol: Chem.Mol) -> dict[int, bool]:
             filtered_mapping[ai] = aj
 
     return filtered_mapping
+
```

Comment:
We should apply `black` to this repo at some point. Probably an issue for @atravitz to deal with.

Reply:
Yes please would love for all the repos to have the same layout!